citi-onboarding/Aqua-Jr
# Feature/post to blog

## What I did
- Created the frontend for the Post to Blog section
## How to test
- Firstly, all changes made locally on your branch should be committed
- Now, access the branch through your terminal:
```  
   git checkout develop
   git pull
   git checkout feature/footer
```
- To continue the testing process, you should now open a second terminal, or, if you're using the VSCode one, just slipt your current one
- On one terminal you should type the following commands to access the client folder and run yarn start
```    
    cd client
    yarn start
```
- If all the previous steps were followed correctly you should now have a blank localhost300 page open on your server
- Moving on, go to your second terminal and access the server folder, where you should run either node index.js or nodemon index.js
```    
    cd server
    nodemon index.js
```
- Now you're all set and should be able to view the websites Post to Blog section on your localhost3000 page. To properly test it, put it through the following tests:

1. Hover over the left arrow and check if it turns blue
2. Click on both arrows and check if each click corresponds to one slide change
3. Change your screen size both randomly and following the sizes suggested by the server

- If the both the web and mobile versions work just as shown in the mockup, everything's fine :)
